_pytest/terminal.py

```diff
@@ -300,10 +300,15 @@ def pytest_report_header(self, config):
         if plugininfo:
             l = []
             for plugin, dist in plugininfo:
-                name = dist.project_name
+                # gets us name and version!
+                name = str(dist)
```

Comment:
It would be nice if this was joined by a dash rather than space, for consistency with the builtin plugins on line 281.